### ISSUE TITLE
Fileset Integrity Check failed

### ISSUE THREAD

**Issue by funkymaster** [Dolibarr/dolibarr#19576]
# Instructions
*This is a template to help you report good issues. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *replace the bracket enclosed texts with meaningful information*
- *remove any unused sub-section*

# Bug
Fileset Integrity Checks fails with missing files.

| # | Filename| Expected size | Expected Checksum |
| --- | --- | --- | --- |
| 1 | /cache.manifest | 1074 Bytes | dcd3eb5cec586de671557315461a21ae |
| 2 | /includes/tecnickcom/tcpdf/fonts/ae_fonts_2.0/ChangeLog.9745.BAK | 29723 Bytes | 81f1d0eb3f38e2ff1cf2c110b620096c |
| 3 | /includes/tecnickcom/tcpdf/fonts/freefont-20120503/ChangeLog.10070.BAK | 432211 Bytes | e80566efc867d2663f4dfd5bae5fd1ba |

The problem is, that the files are not delivered with the default package.

## Environment
- **Version**: 14.0.4
- **OS**: Linux 5.10.0-9-amd64 SMP Debian 5.10.70-1 (2021-09-30) x86_64
- **Web server**: Apache/2.4.51 (Debian)
- **PHP**: 7.4.25
- **Database**: MySQL or MariaDB 5.5.5-10.5.12-MariaDB-0+deb11u1

## Expected and actual behavior
I don't know if the files are really missing.
Either don't check them it they are not necessary or deliver them with the package if they are really needed.

## Steps to reproduce the behavior
- Download Dolibarr:
  - https://www.dolibarr.org/files/stable/standard/dolibarr-14.0.4.zip
  - https://sourceforge.net/projects/dolibarr/files/Dolibarr%20ERP-CRM/14.0.4/dolibarr-14.0.4.zip
- Unzip
- Install/Update
- Start Check: http://x.x.x.x/dolibarr/htdocs/admin/system/filecheck.php?mainmenu=home&leftmenu=admintools_info

## [Attached files](https://help.github.com/articles/issue-attachments) (Screenshots, screencasts, dolibarr.log, debugging informations…)
[*Files*]

**Comment by onwebleon** [comment]
Hi same for me!

### BUG

Fileset Integrity Checks fails with
- 1 missing /includes/nnnick/chartjs/bower.json
- 127 modified (list in attached files)

### Environment

- Version: 14.0.4
- OS: Linux 4.4.59+ aarch64
- Web server: Apache/2.4.43 (Unix)
- PHP: 7.4.9
- Database: MariaDB 5.5.5-10.3.29-MariaDB

### Expected and actual behavior

After fresh install I am expecting to have the integrity check pass
Actually got the error & 1 missing file + 127 modified...

### Steps to reproduce the behavior

1. Have your web server & DB ready
2. Git clone the Dolibar v14 from here
3. Do the setup
4. Run the integrity check

### Attached files

[IntegrityCheck_modifiedFilesListing.csv](https://github.com/Dolibarr/dolibarr/files/7697873/IntegrityCheck_modifiedFilesListing.csv)

**Comment by eldy** [comment]
Can you provide the name of the package you used to install v14.0.4 ?
If you installed with git did you user the tag 14.0.4 (and not the branch v14.0) ?
The 2 .BACK files were removed from the src repository.

**Comment by funkymaster** [comment]
As mentioned above, I used those two packages:
- https://www.dolibarr.org/files/stable/standard/dolibarr-14.0.4.zip
- https://sourceforge.net/projects/dolibarr/files/Dolibarr%20ERP-CRM/14.0.4/dolibarr-14.0.4.zip

I tested both in a fresh new installation.
In any case, all 3 files are really missing on disk.
So the file check seems to be correct.
I get the same results with local and remote signature check.

If the two *.BACK files have been removed, I suggest to remove them from the filelist-14.0.4.xml signature.

What about cache.manifest?
Is it missing or should it also be removed from the filelist-14.0.4.xml signature?
